### DISCUSSION TITLE
Edit any object

### DISCUSSION THREAD

**Discussion by svken** [streetcomplete/StreetComplete#5058] [Ideas]
I think since you have the best user experience in your app it would be great for mappers to enable a "professional mode" where they can edit every object and there attributes manually.

**Comment by mnalis** [comment]
There is [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) _"StreetComplete Expert Edition"_ fork of StreetComplete, which provides exactly that functionality (among other things, see its github page https://github.com/Helium314/SCEE)

(there are also related issues/discussions like https://github.com/streetcomplete/StreetComplete/discussions/3003 and https://github.com/streetcomplete/StreetComplete/issues/471 if you're interested in the history and reasoning why it is not present in upstream StreetComplete - _short spoiler: because it is tailored toward newbies without OSM experience_)
